### DISCUSSION TITLE
How to remove some fields in 5.0 SunbirdEd version?

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#614] [General Discussion]
Hello all,

We're on **4.6 version of SunbirdEd** and now we're seeing some new fields when we installed **5.0 version of SunbirdEd**.

![image](https://github.com/Sunbird-Ed/Community/assets/107101132/b859fe63-39cb-42da-b359-911d8269c698)

As on 4.6 version, the users are not having DOB.
I want to remove this DOB field, how I can achieve that?

@rajeevsathish @Krishnaj20

**Comment by Krishnaj20** [comment]
the year of birth is an important attribute for the system to know whether the user is a minor or major. Accordingly the workflows, specifically where PII information is involved are rendered.

**Reply by nidhikachhawa24** [reply]
@Krishnaj20 We've around 20k users and we can't ask each of them to update DOB, so how can we update DOB of all users?

**Reply by Krishnaj20** [reply]
@rajeevsathish @swayangjit - can you help and provide information around how in DIKSHA we were able to send auto pop ups to already registered users to complete their DoB.

**Reply by rajeevsathish** [reply]
@nidhikachhawa24 
we have added a pipup so that we can take the YOB of the login users 
here is the code for the same
`<app-modal-wrapper [config]="{disableClose: true, size: 'small', panelClass: 'material-modal'}"
      *ngIf="isCustodianOrgUser && usersProfile && !usersProfile?.managedBy && !usersProfile?.dob && (usersProfile?.userLocations?.length>0 || showYearOfBirthPopup)">
      <ng-template sbModalContent let-data>
        <app-year-of-birth [dialogProps]="data"></app-year-of-birth>
      </ng-template>
    </app-modal-wrapper>`
this will be only for a login user and who do not have DOB filled.

cc: @Krishnaj20 @swayangjit

**Reply by Krishnaj20** [reply]
thanks @rajeevsathish . 
Nidhi - if your query is resolved pls close the thread.

**Comment by siddhishinde0723** [comment]
> the year of birth is an important attribute for the system to know whether the user is a minor or major. Accordingly the workflows, specifically where PII information is involved are rendered.

@rajeevsathish @Krishnaj20 @swayangjit    what are the workflows involved and how the behaviour / flow changes for them for major / minor ?
